packages/xo-web/src/xo-app/hub/recipes/recipe-form.js

```diff
@@ -100,6 +108,78 @@ export default decorate([
           value={value.nbNodes}
         />
       </FormGrid.Row>
+      <FormGrid.Row>
+        <label>
+          {_('recipeStaticIpAddresses')}
+          <input
+            className='form-control'
+            name='staticIpAddress'
+            onChange={effects.toggleStaticIpAddress}
+            type='checkbox'
+            value={value.staticIpAddress}
+          />
+        </label>
+      </FormGrid.Row>
+      {value.staticIpAddress && (
```

Comment:
To avoid repeating the verification of this condition, you can use the following syntax
`{condition && [<Component1/>,  <Component2/>]}`